### ISSUE TITLE
Running apps fail needing google services plugin

### ISSUE THREAD

**Issue by joeyda3rd** [tadaspetra/getx_examples#4]
I'm new to app dev, so pardon my ignorance. It looks like your code has configuration for google services in all the apps, even the ones that don't use it, so your code won't build on my setup out of the box. See the error I receive for every app in this repo below, Not a big deal for me, just wanted to let you know for your audience. The easiest way to solve I've found is ~if you rebuild a new flutter app and copy over lib directory and pubspec.yaml it will run.~ to remove the references to google services in the build.gradle files. I haven't run any apps that use firebase yet (looks like the todo app does), but I'm certain there will be some configuration for everyone's firebase account needed there. If it's worth it, maybe remove the references in the build.gradle files where necessary in your branch.

>Plugin project :firebase_core_web not found. Please update settings.gradle.
Plugin project :cloud_firestore_web not found. Please update settings.gradle.
FAILURE: Build failed with an exception.
>* What went wrong:
Execution failed for task ':app:processDebugGoogleServices'.
File google-services.json is missing. The Google Services Plugin cannot function without it. `

**Comment by tadaspetra** [comment]
I think this happens when you try to build for web but the project is not compatible with web. Are you using flutter beta channel by any chance? 

If you are, I would try to switch to the stable channel and then do a "flutter clean" and then it might work

**Comment by joeyda3rd** [comment]
No, I'm on stable 1.22, but I haven't tried building for web yet, just android at this point. If I add the firebase packages and include my google services json file, it builds no problem too.
